### PR TITLE
Created this-week-in-os-19

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Created file
- Partially added missing links from config
- Removed Ricardo from config

### CODE REVIEW

File: src/posts/2022-11-21-this-week-in-os-19.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  function from being run in fastboot ([@mansona])
+- [Turbo87/renovate-config]
+  [#27](https://github.com/Turbo87/renovate-config/pull/27) renovate: Use weekly
+  update schedule ([@Turbo87])
+- [Turbo87/renovate-config]
+  [#10](https://github.com/Turbo87/renovate-config/pull/10) Use Jest to test all
+  relevant JSON files in the project ([@Turbo87])
+- [Turbo87/renovate-config]
+  [#9](https://github.com/Turbo87/renovate-config/pull/9) Add
+  `githubActions/regexManagers` preset ([@Turbo87])
```
Comment (Turbo87) [inline src/posts/2022-11-21-this-week-in-os-19.md]: these can be ignored

File: src/posts/2022-11-21-this-week-in-os-19.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  `test` script to only run `ember test` ([@Turbo87])
+- [mainmatter/ember-api-actions]
+  [#8](https://github.com/mainmatter/ember-api-actions/pull/8) CI: Pin action
+  versions ([@Turbo87])
+- [mainmatter/ember-api-actions]
+  [#4](https://github.com/mainmatter/ember-api-actions/pull/4) Adjust Renovate
+  setup ([@Turbo87])
+- [mainmatter/ember-api-actions]
+  [#2](https://github.com/mainmatter/ember-api-actions/pull/2) CI: Migrate to
+  pnpm ([@Turbo87])
```
Comment (Turbo87) [inline src/posts/2022-11-21-this-week-in-os-19.md]: they are all Ember.js category

File: src/posts/2022-11-21-this-week-in-os-19.md (hunk 1 above)
Comment (emmasofiah2o) [reply src/posts/2022-11-21-this-week-in-os-19.md]: Just this time or in general? Like should I put them in the "exclude" category? :)

Comment (Turbo87) [reply src/posts/2022-11-21-this-week-in-os-19.md]: >  Like should I put them in the "exclude" category?

yep 👍

File: src/posts/2022-11-21-this-week-in-os-19.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+  re-exports ([@nickschot])
+- [mainmatter/ember-cli-list-addons]
+  [#1](https://github.com/mainmatter/ember-cli-list-addons/pull/1) Initial
+  implementation ([@mansona])
+- [mainmatter/eslint-plugin-emberobserver]
+  [#3](https://github.com/mainmatter/eslint-plugin-emberobserver/pull/3) Fix
+  blocker preventing ember-observer from tracking an addon's embroider
+  compatibility ([@candunaj])
+- [mansona/ember-try] [#1](https://github.com/mansona/ember-try/pull/1)
+  onlyVersionCompatibility to generateConfig ([@candunaj])
```
Comment (Turbo87) [inline src/posts/2022-11-21-this-week-in-os-19.md]: they are all "Ember.js" AFAICT

**Pull request merged**